src/main/drivers/accgyro/accgyro_mpu.h

```diff
@@ -156,11 +156,7 @@ enum icm_high_range_gyro_fsr_e {
     NUM_ICM_HIGH_RANGE_GYRO_FSR
 };
 
-enum fchoice_b {
-    FCB_DISABLED = 0x00,
-    FCB_8800_32 = 0x01,
-    FCB_3600_32 = 0x02
-};
+#define FCB_DISABLED 0x00
```

Comment:
Not sure if there's value in keeping this - or-ing it does not change things, and `FCB` isn't indicative enough to tell somebody looking at some years down the line what it meant.

Reply:
Fair enough. 4fdee6e